CarRentalCompany.Domain/Models/CarReceiptForm.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace CarRentalCompany.Domain.Models
+{
+    public class CarReceiptForm
+    {
+        public CarReceiptForm(string type, Guid clientId)
+        {
+            Id = Guid.NewGuid();
+            Type = type;
+            ClientId = clientId;
+            Activities = [];
+        }
+
+        public CarReceiptForm(Guid id, string type, Guid clientId)
+        {
+            Id = id;
+            Type = type;
+            ClientId = clientId;
+            Activities = [];
+        }
+
+        public Guid Id { get; }
+        public string Type { get; }
+        public Guid ClientId { get; }
+        public List<ActivityInstance> Activities { get; }
```

Comment:
Hold: to powinno być jako IReadOnlyList na zewnatrz widoczne. nie chcesz, zeby ktos Ci dodal z zewnątrz jakiś element do listy.